scripts/install-ogdc.sh

```diff
@@ -29,5 +29,10 @@ helm repo add minio https://charts.min.io/
 helm repo add bitnami https://charts.bitnami.com/bitnami
 helm dependency build helm/
 
+RELEASE_NAME="qgnet-ogdc"
+echo "Using RELEASE_NAME=${RELEASE_NAME}"
+QGNET_WORKFLOW_PVC_NAME="${RELEASE_NAME}-workflow-pvc"
+echo "Using QGNET_WORKFLOW_PVC_NAME=${QGNET_WORKFLOW_PVC_NAME}"
+
 # `qgnet-ogdc` is the "release name".
-helm install --set ENV="$ENV" --set OgdcPVHostPath="$OGDC_PV_HOST_PATH" qgnet-ogdc "$THIS_DIR/../helm" -n qgnet
+helm install --set ENV="$ENV" --set minio.persistence.existingClaim=$QGNET_WORKFLOW_PVC_NAME --set QGNetWorkflowPVCName="$QGNET_WORKFLOW_PVC_NAME" --set OgdcPVHostPath="$OGDC_PV_HOST_PATH" $RELEASE_NAME "$THIS_DIR/../helm" -n qgnet
```

Comment:
I wanted to use `{{ .Release.Name }}-workflow-pvc`, as we did in the PVC definition, but that doesn't seem to be possible in the values.yaml.

This creates a new variable based on the release name we use and pass those to helm directly via `--set`.